src/sections/Projects/SMP/smp.style.js

```diff
@@ -1,12 +1,11 @@
 import styled from "styled-components";
-import smp_bg from "./smp-hero.png";
-import callout from "./callout.png";
-import cncfIcon from "./cncf-icon.png";
+const smp = "./smp-hero.png";
```

Comment:
The banners are not displaying these images. These imports can be left as it is right?

Reply:
Yes I think so the variable we use on the `smp.style.js` page needs to be imported. So I think using import here will be better option.